app/assets/javascripts/students/components/CongratsModal.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import Modal from 'react-bootstrap-modal'
+import { RailsRoutes } from '../../shared/routes'
+import { Images, convertImage } from '../../utils/image_helpers'
+
+class CongratsModal extends React.Component {
+  render() {
+    return(
+      <div>
+        <Modal
+          show={this.props.openModal}
+          onHide={this.props.closeModal}
+        >
+          <Modal.Header>
+            <Modal.Title
+              className='congrats-header'>
+              Congratulations!
+            </Modal.Title>
+            <Modal.Dismiss
+              className='flex center modal-dismiss-container'
+              onClick={this.closeModal}>
+              <img
+                src={Images.close_modal}
+                className='modal-dismiss'/>
+            </Modal.Dismiss>
+          </Modal.Header>
+          <Modal.Body>
+            <div>
+              Well done! You have successfully completed this Module!
```

Comment:
i think ken talked to you about this, but its a better idea to have a generic `Modal` component that can display any type of content. or, if you guys are only using response modals in this format, you could pass in a `title` and `text` as props, and use those props to display the generic modal. alternatively, you could pass in a child component instead of a `text` prop so if you need to display something (e.g. image or video) you could do so. in general, try to avoid hardcoding things like this and try to think more flexibly when you are designing components